### ISSUE TITLE
Incomplete ratio groups should still populate

### ISSUE THREAD

**Issue by dmbyte** [SUSE/DeepSea#519]
Currently, if a system has 22 drives, 5 SSDs, and a ratio set to five, the final SSD and 2 drives are unused.  They should be used by default with an option to disable partial ratio drive groupings.

**Comment by swiftgist** [comment]
The runner does have a leftovers option:

```
    leftovers=False - Set to True to propose leftover drives as
                                 standalone OSDs.
```

but I don't think that's what you are asking.  These standalone OSDs are not used by default but can be enabled.

I suspect you are asking for the last group to be two OSDs using the last SSD.

**Comment by dmbyte** [comment]
You are correct in what I am asking is to have the remaining used in a group where the ratio is not fully filled out.  The expectation being that more drives could be added later, resulting in the group being completed.
